test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -135,6 +148,8 @@ public void AutoFitRankingTest()
             var trainDataView = reader.Load(new MultiFileSource(DatasetUtil.GetMLSRDataset()));
             var testDataView = mlContext.Data.TakeRows(trainDataView, 500);
             trainDataView = mlContext.Data.SkipRows(trainDataView, 500);
+            if (!isUsingTrainValidateRunner)
```

Comment:
Maybe we can remove this if condition so this function only run train-validation?